src/ledger/address.ts

```diff
@@ -31,9 +31,14 @@ export class Address implements ABIValue {
 	asABI(): any {
 		return this.toString();
 	}
+
 	ABIType(): string {
 		return "address";
 	}
+
+  isZero(): boolean {
+    return this.value.find(v => v>0) == undefined;
```

Comment:
Although it works here, it is generally advised to always use the strict `===` comparison operator to avoid unexpected outcomes because of implicit typeconversions.

On another note, are those spaces or tabs used? We have a `.prettier` and `.eslintrc` in the repository which can be used with an autoformatter to avoid this (:

Reply:
sorry, using prettier now.